Util.js

```diff
@@ -21,20 +25,25 @@ function removeTrailingSlash_ (string) {
   const length = string.length
   if (string.charAt(length - 1) === '/') {
     // Remove trailing slash
-    return string.substr(0, length - 1)
-  } else {
-    return string
+    string = string.substr(0, length - 1)
   }
+  return string
 }
 
-function getObjectFromResponse_ (response) {
-  return JSON.parse(response.getContentText())
+function fetchObject_ (url, options) {
```

Comment:
I love this!

Reply:
Yeah, it gets even better in the other PR. :)